### PR TITLE
chore(Colors): use new Discord colors

### PR DESCRIPTION
**Pull request by Dev-CasperTheGhost**
**Please describe the changes this PR makes and why it should be merged:**

This PR updates the `Colors` constant to use the new [Discord colors](https://cdn.discordapp.com/attachments/613427779723067411/842438442880073748/2021_Palette_.png). 

**Status and versioning classification:**

- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating

<details>
	<summary><strong>Edit: example of color in an embed</strong></summary>

image

</details>

### CODE REVIEW

**Comment by vBread** [comment]
Should the old blurple be added as `OLD_BLURPLE`/`OG_BLURPLE` if users still prefer it?

**Comment by Dev-CasperTheGhost** [comment]
> Should the old blurple be added as `OLD_BLURPLE`/`OG_BLURPLE` if users still prefer it?

yeah, I could do that. Tho idk what feedback of the maintainers is

**Review by CertifiedRice — APPROVED** [review]

**Comment by Dev-CasperTheGhost** [comment]
> Also, what about Fuchsia?

Whoops missed that one. Will add it now

File: typings/index.d.ts [hunk 1]
```diff
@@ -557,14 +557,15 @@ declare module 'discord.js' {
       DEFAULT: 0x000000;
       WHITE: 0xffffff;
```
Comment (DaStormer) [inline typings/index.d.ts:558]: Add Discord's black?
```suggestion
      WHITE: 0xffffff;
      BLACK: 0x23272a;
```

File: src/util/Constants.js [hunk 2]
```diff
@@ -521,14 +521,15 @@ exports.Colors = {
   DEFAULT: 0x000000,
   WHITE: 0xffffff,
```
Comment (DaStormer) [inline src/util/Constants.js:522]: Add Discord's black?
```suggestion
  WHITE: 0xffffff,
  BLACK: 0x23272a,
```

File: typings/index.d.ts (hunk 1 above)
Comment (Dev-CasperTheGhost) [reply typings/index.d.ts:558]: The black color didn't change I think because it's the same as `NOT_QUITE_BLACK`

Comment (DaStormer) [reply typings/index.d.ts:562]: Ah, I see. Didn't notice that 😅 

Maybe it should be changed to `BLACK` so it's a more straightforward 🤔 just a thought though

Comment (Dev-CasperTheGhost) [reply typings/index.d.ts:562]: I think we can keep as is for now and see what the maintainers think 😄

Comment (monbrey) [reply typings/index.d.ts:562]: The new BLACK on the branding guidelines is #000000.

The old "black" can stay as it is

**Review by catcosmicice — APPROVED** [review]

**Comment by sasial-dev** [comment]
> > Should the old blurple be added as `OLD_BLURPLE`/`OG_BLURPLE` if users still prefer it?
> 
> yeah, I could do that. Tho idk what feedback of the maintainers is

I'm not a maintainer ofc, but I would love to see it!

**Comment by iCrawl** [comment]
We have no interest of adding OLD_BLURPLE as an available color. It does not reflect Discord's color palette and design system anymore.

This might be an unpopular opinion/change but since this is for v13 anyway there is really no need to provide a "backwards compatible" (since it's not really one) way for the old color. Those who wish to use it can just save the value locally somewhere.

**Review by kyranet — CHANGES_REQUESTED** [review]
Please address @iCrawl's feedback.

On an additional note, if Discord were to change colors again, how would we name the old keys? `OLD_OLD_BLURPLE`? For this reason, versioning old/unused colors from them seems like a bad idea.

**Comment by Dev-CasperTheGhost** [comment]
> Please address iCrawl's feedback.

What do you mean? This PR does not include any old colors, unless I'm mistaken

**Review by SpaceEEC — APPROVED** [review]

**Comment by almostSouji** [comment]
isn't this changing non-discord colors as well? I don't believe the pre-PR yellow was a discord branding color but just the most saturated yellow variation as per web standards

**Comment by kyranet** [comment]
Oops, sorry, I mixed PRs up, today has been a long day @Dev-CasperTheGhost.

**Review by kyranet — APPROVED** [review]

**Comment by Dev-CasperTheGhost** [comment]
> Oops, sorry, I mixed PRs up, today has been a long day @Dev-CasperTheGhost.

No worries!

**Review by iCrawl — APPROVED** [review]

**Pull request merged**